docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -325,36 +351,37 @@ Re-Configuring ArduCopter DSHOT Outputs
 applies to the popular Cube Orange flight controller.
 
-The MAIN outputs of these flight controllers can put out PWM, but not DSHOT. One way to check for this issue is to check the "Messages" section of the "Data" tab in Mission Planner on reboot.
+For affected flight controllers their main outputs can put out PWM, but not DSHOT. One way to check for this issue is to check the "Messages" section of the "Data" tab in Mission Planner on reboot.
 This will display what kinds of protocols are actually going to be output on each pin based on the configuration. Set your flight controller for DSHOT as discussed above, reboot it, connect to it,
-and check the "Messages" section. If you see a message like the one below that lists "RCOut: PWM 1-12", that means your outputs are only going to put out PWM still, and you need to move the motor to
-a DSHOT compatible output.
+and check the "Messages" section. If you see a message like the one below that lists "RCOut: PWM 1-12", that means your flight controller outputs will still only output PWM, and you must connect the module to
+a DSHOT compatible output instead.
 
 .. figure:: ../_static/tutorial_images/pwm_flight_controller/mp_messages_pwm.JPG
     :align: center
 
     Message Showing Outputs are PWM Only
 
-You need to move the motor output from a MAIN output to an AUX output. In this case, we will move from MAIN OUT 1 to AUX OUT 1. The physical connection of the signal and ground wire to the flight 
-controller is shown below.
+You need to move the module from a main output to an AUX output (these may also be labeled as FMU PWM outputs). For the Pix32 used to test this example, the module was moved from MAIN OUT 1 to AUX OUT 1. 
+Exactly which pins to use and how they are labeled will vary depnding on your flight controller, refer to the flight controller's documentation for more information. Newer Pixhawk models may label these
+outputs as FMU PWM OUT insetad of AUX OUT. The physical connection of the signal and ground wire to the flight controller for the Pix32 is shown below.
 
 .. figure:: ../_static/tutorial_images/pwm_flight_controller/dshot_aux.jpg
     :align: center
     :width: 60%
 
-    Connecting to AUX 1 OUT
+    Connecting to AUX OUT 1
 
-Next, you need to tell ArduCopter to use AUX 1 as the output for Motor 1 on your vehicle, instead of the default of MAIN 1. ArduCopter uses SERVOX_FUNCTION variables to assign a function to each
-output. For the Pixhawk, AUX 1 is controlled by SERVO9_FUNCTION. **Set SERVO9_FUNCTION = 33 to output the throttle commands for Motor 1 on AUX 1.** See the ArduCopter documentation for details on what
-settings to use for different functions. The figure below shows the proper setting for this parameter.
+Next, you need to tell ArduCopter to use AUX OUT 1 or whatever the equivalent pin on your flight controller is as the output for Motor 1 on your vehicle. ArduCopter uses SERVOX_FUNCTION variables to assign a function to each
```

Comment:
Next, you need to tell ArduCopter to use AUX OUT 1 (or the equivalent pin on your flight controller) as the ...